junit/MatrixTest.java

```diff
@@ -437,6 +437,16 @@ public void testGetException()
         testIndex[1] = -1; 
         matrix.get(testIndex[0], testIndex[1]);
     }
+    /**
+     * testRandom -Tests the generation of a random matrix
+     *  of the specified dimensions. 
+     */
+    @Test
+    public void testRandom() 
+    {
+        //TODO: How to seed random number gen across class instances?
+        Random rand = new Random();
```

Comment:
We could add a static class member that hold the RNG and have getter/setter methods to access it. That way, we could set its seed before creating the matrix and predict its output. That being said, I'm not sure of the "worth-it" factor of doing that. Thoughts @bclinthall ?